packages/calling-component-bindings/src/handlers/createACSCallHandlers.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Call, CallAgent, StartCallOptions } from '@azure/communication-calling';
+/* @conditional-compile-remove(dialpad) */ /* @conditional-compile-remove(PSTN-calls) */
+/* @conditional-compile-remove(PSTN-calls) */
+import { AddPhoneNumberOptions } from '@azure/communication-calling';
+/* @conditional-compile-remove(PSTN-calls) */
+import {
+  isCommunicationUserIdentifier,
+  isMicrosoftTeamsUserIdentifier,
+  isPhoneNumberIdentifier
+} from '@azure/communication-common';
+import { CommunicationIdentifier } from '@azure/communication-common';
+
+import { _toCommunicationIdentifier } from '@internal/acs-ui-common';
+import { StatefulCallClient, StatefulDeviceManager } from '@internal/calling-stateful-client';
+import memoizeOne from 'memoize-one';
+import { isNonTeamsCallParticipants } from '../utils/callUtils';
+import { CallingHandlersCommon, createDefaultCallingHandlersCommon } from './createHandlersCommon';
+
+/**
+ * Object containing all the handlers required for calling components.
+ *
+ * Calling related components from this package are able to pick out relevant handlers from this object.
+ * See {@link useHandlers} and {@link usePropsFor}.
+ *
+ * @public
+ */
+export type CallingHandlers = Omit<CallingHandlersCommon, 'onStartCall'> & {
+  onStartCall: (participants: CommunicationIdentifier[], options?: StartCallOptions) => Call | undefined;
+};
+
+/**
+ * @public
+ */
+export const createDefaultACSCallingHandlers = memoizeOne(
+  (
+    callClient: StatefulCallClient,
+    callAgent: CallAgent | undefined,
+    deviceManager: StatefulDeviceManager | undefined,
+    call: Call | undefined
+  ): CallingHandlers => {
+    return {
+      ...createDefaultCallingHandlersCommon(callClient, deviceManager, call),
+      // FIXME: onStartCall API should use string, not the underlying SDK types.
+      onStartCall: (participants: CommunicationIdentifier[], options?: StartCallOptions): Call | undefined => {
+        if (!isNonTeamsCallParticipants(participants)) {
+          throw new Error('TeamsUserIdentifier in Teams call is not supported!');
+        }
+        return callAgent ? callAgent.startCall(participants, options) : undefined;
+      },
+      /* @conditional-compile-remove(PSTN-calls) */
+      onAddParticipant: async (
+        userId: string | CommunicationIdentifier,
+        options?: AddPhoneNumberOptions
+      ): Promise<void> => {
+        const participant = _toCommunicationIdentifier(userId);
+        if (isPhoneNumberIdentifier(participant)) {
```

Comment:
Again, I don't understand why we need to stop forwarding `options` to the headless API in some cases, given that the type of `options` is `AddPhoneNumberOptions`.

Reply:
I believe this is a copy&paste code, lemme see if I can forward this param

Reply:
![image](https://user-images.githubusercontent.com/11863655/200987243-902b1388-bdd4-441b-805c-da5cf7354940.png)
Just checked api we are exposing in PSTN calling feature, looks like we don't currently expose options for all the other types of identifier, so I am not changing any logic here for now